content/en/security/application_security/enabling/serverless.md

```diff
@@ -216,9 +219,825 @@ To install and configure the Datadog Serverless Framework plugin:
+   ```dockerfile
+   ENTRYPOINT ["/app/datadog-init"]
+   ```
+
+5. Execute your binary application wrapped in the entrypoint, launched by the Datadog trace library. Adapt this line to your needs.
+   ```dockerfile
+   CMD ["/dd_tracer/python/bin/ddtrace-run", "python", "app.py"]
+   ```
+#### Alternative configuration {#alt-python}
+If you already have an entrypoint defined inside your Dockerfile, you can instead modify the CMD argument.
+
+{{< highlight dockerfile "hl_lines=7" >}}
+COPY --from=datadog/serverless-init:1 /datadog-init /app/datadog-init
+RUN pip install --target /dd_tracer/python/ ddtrace
+ENV DD_SERVICE=datadog-demo-run-python
+ENV DD_ENV=datadog-demo
+ENV DD_VERSION=1
+ENV DD_APPSEC_ENABLED=1 # Optional: if you want to enable ASM
+CMD ["/app/datadog-init", "/dd_tracer/python/bin/ddtrace-run", "python", "app.py"]
+{{< /highlight >}}
+
+If you require your entrypoint to be instrumented as well, you can swap your entrypoint and CMD arguments instead. For more information, see [How `serverless-init` works](#how-serverless-init-works).
+
+{{< highlight dockerfile "hl_lines=7-8" >}}
+COPY --from=datadog/serverless-init:1 /datadog-init /app/datadog-init
+RUN pip install --target /dd_tracer/python/ ddtrace
+ENV DD_SERVICE=datadog-demo-run-python
+ENV DD_ENV=datadog-demo
+ENV DD_VERSION=1
+ENV DD_APPSEC_ENABLED=1 # Optional: if you want to enable ASM
+ENTRYPOINT ["/app/datadog-init"]
+CMD ["your_entrypoint.sh", "/dd_tracer/python/bin/ddtrace-run", "python", "app.py"]
+{{< /highlight >}}
+
+As long as your command to run is passed as an argument to `datadog-init`, you will receive full instrumentation.
+
+[1]: /tracing/trace_collection/dd_libraries/python/?tab=containers#instrument-your-application
+
+{{% /tab %}}
+{{% tab "Java" %}}
+
+Add the following instructions and arguments to your Dockerfile.
+
+```dockerfile
+COPY --from=datadog/serverless-init:1 /datadog-init /app/datadog-init
+ADD https://dtdg.co/latest-java-tracer /dd_tracer/java/dd-java-agent.jar
+ENV DD_SERVICE=datadog-demo-run-java
+ENV DD_ENV=datadog-demo
+ENV DD_VERSION=1
+ENV DD_APPSEC_ENABLED=1 # Optional: if you want to enable ASM
+ENTRYPOINT ["/app/datadog-init"]
+CMD ["./mvnw", "spring-boot:run"]
+```
+
+If you require your entrypoint to be instrumented as well, you can swap your entrypoint and CMD arguments instead. To understand why this may be required, please refer to [how serverless-init works](#how-it-works)
+
+{{< highlight dockerfile "hl_lines=7-8" >}}
+COPY --from=datadog/serverless-init:1 /datadog-init /app/datadog-init
+COPY --from=datadog/dd-lib-js-init /operator-build/node_modules /dd_tracer/node/
+ENV DD_SERVICE=datadog-demo-run-nodejs
+ENV DD_ENV=datadog-demo
+ENV DD_VERSION=1
+ENV DD_APPSEC_ENABLED=1 # Optional: if you want to enable ASM
+ENTRYPOINT ["/app/datadog-init"]
+CMD ["/your_entrypoint.sh", "/nodejs/bin/node", "/path/to/your/app.js"]
```

Comment:
This is within the Java section but seems to be a nodejs example?

Reply:
Verified with Serverless docs writer that this was incorrect in the Serverless - Google Cloud Run docs, I have pushed a [PR](https://github.com/DataDog/documentation/pull/20248) to remove that section from those docs and will remove it here as well.